File: MPI/mpicheck/tests/functions/2.0/MPI_Win_set_attr.f90

        
        program main
        use mpi
        
        INTEGER var_0
       INTEGER var_1
       INTEGER(KIND=MPI_ADDRESS_KIND) var_2
       INTEGER var_3
        call mpi_win_set_attr(var_0, var_1, var_2, var_3)
       call pmpi_win_set_attr(var_0, var_1, var_2, var_3)
        end program main
    

File: MPI/mpicheck/tests/functions/2.0/MPI_Win_set_attr.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    MPI_Win var_0;
    int var_1;
    void *var_2;
    int ret;
    /* calls */
    ret = MPI_Win_set_attr(var_0, var_1, var_2);
    ret = PMPI_Win_set_attr(var_0, var_1, var_2);
    return 0;
}
